src/Payments.sol

```diff
@@ -35,15 +37,16 @@ interface IValidator {
 
 // @title Payments contract.
 contract Payments is ReentrancyGuard {
+    using Dutch for uint256;
     using SafeERC20 for IERC20;
     using RateChangeQueue for RateChangeQueue.Queue;
 
     // Maximum commission rate in basis points (100% = 10000 BPS)
     uint256 public constant COMMISSION_MAX_BPS = 10000;
-    uint256 private constant AUCTION_START_PRICE = .083 ether; // 0.083 FIL
+    uint88 private constant AUCTION_START_PRICE = 10.4 ether; // 10.4 FIL
```

Comment:
where did we pick the 10.4 FIL value? if we consider 300TiB network size and PDP + Payment fee (and payment fee being implemented with the 0.5% of the volume, with the minimum fee value), then it should be something like 7.8 i think

Reply:
Great. This value assume a network size of ~10 TiB, we need to decide what is the starting point we want to set up (whether it is the target network size for having 50USDFC transacted every day, i.e. 300 TiB, or a smaller initial size of 10TiB)

Reply:
Increasing this to 7.83×4=31.32 FIL per our discussion. It's slightly safer to start high.